js/runner.go

```diff
@@ -170,6 +170,10 @@ func (r *Runner) newVU(id int64, samplesOut chan<- stats.SampleContainer) (*VU,
 		BlockedHostnames: r.Bundle.Options.BlockedHostnames.Trie,
 		Hosts:            r.Bundle.Options.Hosts,
 	}
+	if r.Bundle.Options.LocalIPs.Valid {
+		dialer.Dialer.LocalAddr = &net.TCPAddr{IP: r.Bundle.Options.LocalIPs.Pool.GetIP(uint64(id - 1))}
```

Comment:
From my testing, id will be 0 in the first VU allocation. so a check of id > 0 should be set.

Reply:
if r.Bundle.Options.LocalIPs.Valid && id > 0 {

Reply:
there is a fact: VU (id==0) will never sends requests... I think this is a bug

Reply:
VU with id 0 is generally used for stuff that "don't" do requests but ... `setup` and `teardown` functions can make them and they need to also use one of the provided IPs to make the requests. 

Actually speaking of this ... maybe they should use some "randomish" one but instead the first one? Currently They will use `(2**64 -1) % IPPool.count` which IMO is fine - they use an IP from the provided ones and arguably every IP should be viable, but maybe they should use the first one instead for more consistancy? 
cc @imiric @na--

Reply:
I don't think we should add any special handling for 0 ID VUs, but we should fix the `uint64` underflow for the `id=0` case. Picking the first one seems fine to me.

Reply:
from my view, take the first IP is not good since the vu id == 0 will take initialising requests - I guess it is DNS resolving?
if skipped, first VU(id == 0) will be using OS interface ip address (OS to decide what interface & its IP address to reach dns servers). this is needed because most of outside dns servers may not have route to responding to k6's --local-ips.